packages/app/src/background/services/credentials/index.ts

```diff
@@ -62,197 +68,216 @@ export default class VerifiableCredentialsService implements IBackupable {
     return VerifiableCredentialsService.INSTANCE;
   }
 
-  addVerifiableCredentialRequest = async (serializedVerifiableCredential: string): Promise<void> => {
-    await validateSerializedVerifiableCredential(serializedVerifiableCredential);
+  /**
+   * Handle a request to add a Verifiable Credential. Opens a popup to confirm the request.
+   * @param serializedVC - The serialized Verifiable Credential to add.
+   * @returns A promise that resolves when the popup is opened.
+   */
+  handleVCRequest = async (serializedVC: string): Promise<void> => {
+    await validateSerializedVC(serializedVC);
     await this.browserController.openPopup({
-      params: { redirect: Paths.ADD_VERIFIABLE_CREDENTIAL, serializedVerifiableCredential },
+      params: { redirect: Paths.ADD_VERIFIABLE_CREDENTIAL, serializedVerifiableCredential: serializedVC },
     });
   };
 
-  rejectVerifiableCredentialRequest = async (): Promise<void> => {
-    await this.notificationService.create({
-      options: {
-        title: "Request rejected",
-        message: `Rejected a request to add Verifiable Credential.`,
-        iconUrl: browser.runtime.getURL("/icons/logo.png"),
-        type: "basic",
-      },
+  /**
+   * Announces rejection of a request to add a Verifiable Credential.
+   * @returns A promise that resolves when the rejection is announced.
+   */
+  rejectVCRequest = async (): Promise<void> => {
+    await this.announce({
+      notificationTitle: "Request rejected",
+      notificationMessage: `Rejected a request to add Verifiable Credential.`,
+      notificationType: "basic",
+      historyOperationType: OperationType.REJECT_VERIFIABLE_CREDENTIAL_REQUEST,
+      responseType: EventName.USER_REJECT,
+      responsePayload: { type: EventName.VERIFIABLE_CREDENTIAL_REQUEST },
     });
-
-    const tabs = await browser.tabs.query({ active: true });
-    await Promise.all(
-      tabs.map((tab) =>
-        browser.tabs
-          .sendMessage(tab.id!, {
-            type: EventName.USER_REJECT,
-            payload: { type: EventName.ADD_VERIFIABLE_CREDENTIAL },
-          })
-          .catch(() => undefined),
-      ),
-    );
   };
 
-  addVerifiableCredential = async (addVerifiableCredentialArgs: IAddVerifiableCredentialArgs): Promise<void> => {
-    const { serializedVerifiableCredential, verifiableCredentialName } = addVerifiableCredentialArgs;
+  /**
+   * Adds a Verifiable Credential to the store.
+   * @param serializedVerifiableCredential - The serialized Verifiable Credential to add.
+   * @param verifiableCredentialName - User defined name for the Verifiable Credential.
+   * @returns A promise that resolves when the Verifiable Credential is added.
+   * @throws An error if the Verifiable Credential is invalid or already exists.
+   */
+  addVC = async ({ serializedVerifiableCredential, verifiableCredentialName }: IAddVCArgs): Promise<void> => {
```

Comment:
```suggestion
  addVC = async ({ serializedVC, vcName }: IAddVCArgs): Promise<void> => {
```

Reply:
I think it's better to user serialized and name because service contains this in the name. This is related to all the method params for this file.

Reply:
Makes sense. Renamed to `serialized` and `name` in all instances